docs/lang/articles/advanced/performance.md

```diff
@@ -66,3 +66,122 @@ def func():
     for i in range(8192):  # no decorator, use default settings
         ...
 ```
+
+# Local Storage Optimizations
```

Comment:
Maybe this can be changed to a smaller font. Same applies to the nested level. I find it a bit confusing to have two first level (#) title in one page.